src/components/side_panel/chart/building_blocks/axis_design/axis_design_editor.ts

```diff
@@ -59,6 +61,18 @@ export class AxisDesignEditor extends Component<Props, SpreadsheetChildEnv> {
     this.props.updateChart(this.props.figureId, { axesDesign });
   }
 
+  updateTitleFontSize(fontSize: number) {
```

Comment:
```suggestion
  updateAxisTitleFontSize(fontSize: number) {
```

nitpick. To match the other method names